control/plugin/execution.go

```diff
@@ -228,3 +228,19 @@ func (e *ExecutablePlugin) captureStderr() {
 		}
 	}()
 }
+
+// logCapturedLine logs a captured stderr/stdout line from a plugin, after choosing the
+// appropriate log level to use based on the line's contents.
+func logCapturedLine(entry *log.Entry, text string) {
+	lc := strings.ToLower(text)
+	switch {
+	case strings.Contains(lc, "error"):
```

Comment:
@cce  I think it is not a good idea to detect log level in this way. 
For example, the message 'warn: cannot collect metric /xx/xx/xx/error_rate from xxx platform' will be an error instead of warning in your case.

Reply:
You're right, I was looking for a quick solution to ensure that a plugin's error logs are not logged at "DEBUG" level. What do you think is the best approach? Could we assume logrus format and look for `level=error`?

Reply:
@cce There's a similar issue already opened.
https://github.com/intelsdi-x/snap/issues/1186

Reply:
@cce I think parsing "level=error" like strings approach is acceptable. Though it will not be perfect (may be confused if this phrase occurred in log message), but the probability is rather low and the cost is not too high.